src/styles/cards.scss

```
@@ -0,0 +1,172 @@
+.page {
+
+  &__catalog {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 72px;
+  }
+}
+
+.catalog {
+
+  &__container {
+    display: grid;
+    margin-top: 50px;
+    column-gap: 48px;
+    row-gap: 46px;
+
+    @media (min-width: 488px) {
+      & {
+        grid-template-columns: repeat(2, 1fr);
+      }
+    }
+
+    @media (min-width: 768px) {
+      & {
+        grid-template-columns: repeat(3, 1fr);
+      }
+    }
+
+    @media (min-width: 1024px) {
+      & {
+        grid-template-columns: repeat(4, 1fr);
+      }
+    }
+
+    @media (max-width: 488px) {
+      & {
+        grid-template-columns: 1fr;
+      }
+    }
+  }
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s ease 0s;
+
+  &__image {
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+    & img {
+      height: 134px;
+      max-width: 100%;
+    }
+  }
+
+  &__info {
+
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    border-radius: 5px;
+    padding: 12px 0;
+    background-color: $color_blue;
+    transition: all 0.3s ease 0s;
+
+    font-weight: 700;
+    color: $color_white;
+    text-decoration: none;
+    font-size: 14px;
+    text-transform: uppercase;
+    border: 1px solid $color_blue;
+
+    &:hover {
+      color: $color_blue;
+      background-color: $color_white;
+    }
+  }
+
+  &__main {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    color: $color_dark-purple;
+    margin: 0;
+    font-size: inherit;
+    font-weight: 500;
+    line-height: 1.5;
```

use `px` instead this
```suggestion
    line-height: ...px;
```